src/chip-typeahead/index.tsx

```diff
@@ -52,11 +52,12 @@ export interface ChipTypeaheadChildren {
 		props: ListItemProperties & MenuItemProperties
 	) => RenderResult;
 	/** Custom renderer for selected items */
-	selected?: (value: string, label?: string) => RenderResult;
+	selected?: (value: string, label: string) => RenderResult;
```

Comment:
Should we change this to `selected?: (options: ListOption) => RenderResult`

Reply:
I think `ListOption` contains `divider` / `disabled` so probably not?